pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go

```diff
@@ -305,6 +305,7 @@ func TestExtractLibInfo(t *testing.T) {
 		pod                  *corev1.Pod
 		containerRegistry    string
 		expectedLibsToInject []libInfo
+		expectedPodEligible  int // using this as a flag system
```

Comment:
Added this to flag _some_ of the tests with `1` or `-1` on whether or not they should validate pod eligibility or not.